### PR TITLE
Update liveVideoHero to manage dynamic min-height

### PR DESCRIPTION
**Pull request by Xb0X**
Here is a fix to remove the min-height when in player mode.

### CODE REVIEW

File: src/modules/LiveVideoHero/index.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ import ProgramDetails from '../../elements/ProgramDetails';
 const StyledLiveVideoHeroWrapper = styled.div`
   position: relative;
 
-  & ::before, ::after {
-    content: "";
+  & ::before,
```
Comment (arturkin) [inline src/modules/LiveVideoHero/index.js]: looks like you're missing correct after selector here:
`& ::before, & ::after`

Comment (Xb0X) [reply src/modules/LiveVideoHero/index.js]: thx ;) it's done !!

File: src/modules/LiveVideoHero/index.js [hunk 2]
```diff
@@ -22,30 +23,33 @@ const StyledLiveVideoHeroWrapper = styled.div`
...
-    min-height: 408px;
-  `)}
-  ${breakpoints.large(css`
-    min-height: 450px;
-  `)}
-  ${breakpoints.wide(css`
-    min-height: 580px;
-  `)}
+  ${props =>
+    !props.videoPlayerMode &&
```
Comment (JulienKode) [inline src/modules/LiveVideoHero/index.js]: Should we add a test or snapshot to test when the `props.videoPlayerMode` is here or not ?

Comment (Xb0X) [reply src/modules/LiveVideoHero/index.js]: we have the snap ;)

**Pull request merged**